os.go

```diff
@@ -222,6 +222,16 @@ func linkCount(f os.FileInfo) string {
 	return ""
 }
 
+func getBlockSize(f os.FileInfo) (*int64, error) {
```

Comment:
This value indicates the number of blocks, not the size of each block. I think a name like `getBlockCount` might be more fitting.

Reply:
Fair, renamed